_data/compose-cli/docker_compose_run.yaml

```diff
@@ -131,6 +148,17 @@ options:
   default_value: '[]'
   description: Publish a container's port(s) to the host.
   deprecated: false
+  hidden: false
+  experimental: false
+  experimentalcli: false
+  kubernetes: false
+  swarm: false
+- option: quiet-pull
```

Comment:
FWIW; on `docker run`, this option is just named `--quiet` (but the flag's description describes what "quiet" means for `docker run`); https://github.com/docker/cli/blob/7b4d7717b1d14dbaf251c343ba47b242d9e72eb6/cli/command/container/run.go#L61

Reply:
I will open a PR to rename to `quiet` and add an alias for `quiet-pull` to keep backward compatibility

Reply:
I just checked and Compose isn't working the same way as the cli, we only have one flag which manage the pull option without displaying the output. 
We don't have the `pull` + another flag to manage the output just the `quiet-pull` one